reviewing/dependencies-hierarchy/src/buildDependenciesHierarchy.ts

```diff
@@ -132,21 +132,19 @@ async function dependenciesHierarchyForPackage (
       })
       let newEntry: PackageNode | null = null
       const matchedSearched = opts.search?.(packageInfo)
+      const packageAbsolutePath = refToRelative(ref, alias)
```

Comment:
Here's what `getPkgInfo` declares `packageAbsolutePath` to be for reference.

https://github.com/pnpm/pnpm/blob/ee081825bfd43abd9d93b88f94657c7fbb75db25/reviewing/dependencies-hierarchy/src/getPkgInfo.ts#L60